gateway/security.yaml

```diff
@@ -9,6 +9,8 @@ georchestra:
           extended: true
           url: ldap://ldap:389/
           baseDn: ${ldapBaseDn:dc=georchestra,dc=org}
+          adminDn: ${ldapAdminDn:cn=admin,dc=georchestra,dc=org"}
+          adminPassword: ${ldapAdminPassword:secret}
```

Comment:
I think there's no need here to have a default value - it does not make sense for a password